resources/views/profile.blade.php

```diff
@@ -9,7 +9,9 @@
         <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
             <div class="bg-white overflow-hidden shadow-sm sm:rounded-lg">
                 <div style="text-align:center;"class="p-6 bg-white border-b border-gray-200">
-                    Acct details
+                    User ID: {{ auth()->user()->id }}</br>
```

Comment:
where did you find this? i can't find what `auth()->user()` actually does except in one stackoverflow post. regardless it doesn't look like you're using the `$user` model you pass in from the controller.